src/remote-config/remote-config-api-client.ts

```diff
@@ -330,6 +389,49 @@ export class RemoteConfigApiClient {
         'ETag header is not present in the server response.');
     }
   }
+
+  /**
+   * Checks if a given `ListVersionsOptions` object is valid. If successful, transforms the options
+   * object in place by converting `startTime` and `endTime` to RFC3339 UTC "Zulu" format, if present.
+   * 
+   * @param {ListVersionsOptions} options An options object to be validated.
+   */
+  private validateListVersionsOptions(options: ListVersionsOptions): void {
+    if (!validator.isNonNullObject(options)) {
+      throw new FirebaseRemoteConfigError(
+        'invalid-argument',
+        'ListVersionsOptions must be a non-null object');
+    }
+    if (typeof options.pageSize !== 'undefined' && !validator.isNumber(options.pageSize)) {
```

Comment:
Also check for bounds. I guess it should be greater than 0 and less than or eq to 300?

Reply:
The REST API ignores 0 and values greater than 300 and returns all the versions. I added a check anyway :)